scripts/create_master_checklist.py

```diff
@@ -98,20 +98,21 @@ def update_excel_file(input_excel_file, output_excel_file, checklist_data):
     row1 = 8        # First row after which the Excel spreadsheet will be updated
     col_checklist_name = "A"
     row_checklist_name = "4"
-    guid_column_index = "L"
-    comment_column_index = "G"
+    guid_column_index = "M"
+    comment_column_index = "I"
     sample_cell_index = 'A4'
     col_checklist="A"
     col_area = "B"
     col_subarea = "C"
-    col_check = "D"
-    col_desc = "E"
-    col_sev = "F"
-    col_status = "G"
-    col_comment = "H"
-    col_link = "I"
-    col_training = "J"
-    col_arg_success = "K"
+    col_waf_pillar = "D"
+    col_check = "E"
+    col_desc = "F"
+    col_sev = "G"
+    col_status = "H"
+    col_comment = "I"
+    col_link = "J"
+    col_training = "K"
+    col_arg_success = "L"

```

Comment:
@erjosito - this is the only line I think really needs to be looked at in case I got it wrong.  I was confused by col_arg_success vs. col_arg_failure since we only have one arg column in the empty spreadsheet.  I set both to the same value.